README.md

```diff
@@ -1 +1,21 @@
-# tso500_combined_tsv_vs_vcf_check
+# TSO500 CombinedVariantOutput tsv vs vcf check
+
+Script to compare VCFs against TSO500 CombinedVariantOutput tsvs files. This is to find any differences in variant calls between the tsv and vcf, and output 2 separate files detailing any variants present only in either the vcf or tsv for a given run of samples.
```

Comment:
I'm assuming its the VCF from the VEP output, could you mention which VCF type is expected or from what app output?

Reply:
since it just matches on chrom, pos, ref and alt it doesn't matter what sort of vcf it is, anything only in the vcf will have the full record returned and output into the `_all_vcf_only.tsv` file